setup.py

```diff
@@ -23,7 +23,11 @@
     packages=["pytest_pylint"],
     entry_points={"pytest11": ["pylint = pytest_pylint.plugin"]},
     python_requires=">=3.7",
-    install_requires=["pytest>=5.4", "pylint>=2.3.0", "toml>=0.7.1"],
+    install_requires=[
+        "pytest>=5.4",
+        "pylint>=2.3.0",
+        "tomli; python_version < '3.11'",
```

Comment:
```suggestion
        "tomli>=1.1.0; python_version < '3.11'",
```

See https://github.com/hukkin/tomli/blob/master/CHANGELOG.md#110